hunting/windows/queries/microsoft_office_child_processes_with_low_occurrence_frequency.toml

```diff
@@ -1,23 +1,27 @@
 [hunt]
 author = "Elastic"
+description = """
+This hunt looks for Microsoft Office child processes with low occurrence frequency. This could be a normal rare behavior as well as potential execution via a malicious document. Adversaries may use Microsoft Office applications to execute malicious code, such as macros, scripts, or other payloads.
+"""
 integration = ["endpoint", "windows", "system"]
 uuid = "74b2e54b-7002-4201-83d6-7fd9bd5dcf0f"
-name = "Microsoft Office Child Processes with low occurrence frequency"
+name = "Microsoft Office Child Processes with Low Occurrence Frequency by Unique Agent"
 language = "ES|QL"
 license = "Elastic License v2"
 notes = [
-    "Certain processes like WerFault.exe, dw20.exe and dwwin.exe are often related to application crash.",
+    "Certain processes like `WerFault.exe`, `dw20.exe` and `dwwin.exe` are often related to application crash.",
     "Closer attention should be attributed to lolbins and unsigned executables (Windows 4688 is not capturing process code signature information).",
 ]
-mitre = [ "T1566", "T1566.001",]
-
-query = '''
+mitre = [ "T1566", "T1566.001"]
+query = [
+'''
 from logs-endpoint.events.process-*, logs-windows.sysmon_operational-*, logs-system.security-*
-| where host.os.family == "windows" and @timestamp > NOW() - 15 day and 
-  event.category == "process" and event.action in ("start", "Process creation", "created-process") and 
+| where host.os.family == "windows" and @timestamp > NOW() - 15 day and
+  event.category == "process" and event.action in ("start", "Process creation", "created-process") and
   process.parent.name.caseless in ("winword.exe", "excel.exe", "powerpnt.exe") and not starts_with(process.executable, "C:\\Program Files")
```

Comment:
I don't think these would work as only Elastic Defend has caseless fields atm

Reply:
using `to_lower` which was added in 8.14